model/src/main/java/org/openremote/model/value/ValueType.java

```diff
@@ -147,7 +147,7 @@ public static class MultivaluedStringMap extends HashMap<String, List<String>> {
 
     public static final ValueDescriptor<Integer> DIRECTION = new ValueDescriptor<>("direction", Integer.class,
         new ValueConstraint.Min(0),
-        new ValueConstraint.Max(359)
+        new ValueConstraint.Max(360)
```

Comment:
Saw this throwing an exception in demo setup with weather API using the value `360` for wind direction so I've made this accept that value.